### ISSUE TITLE
hasMany directive is ignoring Anonymous Global Scopes on model

### ISSUE THREAD

**Issue by trasteroandy** [nuwave/lighthouse#319]
Hello,

I have this line in a graphql type:

```
cuentas: [Cuenta]! @hasMany
```

and the Laravel Model "Cuenta" has declared an [Anonymous Global Scope](https://laravel.com/docs/5.7/eloquent#query-scopes):

```
class Cuenta extends Model
{
    protected static function boot()
    {
        parent::boot();
        static::addGlobalScope('habilitado', function (Builder $builder) {
            $builder->where('habilitado', '=', true);
        });
    }
```

It works fine in normal code in laravel, but when the client (Apollo) calls the graphql server, the response is using some query that **ignores the global scope** "habilitado".

**Expected behavior**

Don't know if LightHouse use the default relations of the model, but the query that LightHouse uses to retrieve "cuentas" should include:
```
where habilitado = true
```
and it doesn't.

**Environment**

Lighthouse Version: 2.1
Laravel Version: 5.6
PHP Version: 7.2

Thanks a lot, best regards

**Comment by spawnia** [comment]
Can you try my fixed branch https://github.com/mll-lab/lighthouse/tree/native-load-with and see if that does it for you?

**Comment by trasteroandy** [comment]
Hi, I have tried that fixed branch, but got the error:

No directive has been registered for [scalar]

becouse the file "lighthouse/src/Schema/Directives/Nodes/*ScalarDirective.*
php" has been removed. I have this code in my schema.graphql definition:

scalar DateTime @scalar(class:
"Nuwave\\Lighthouse\\Schema\\Types\\Scalars\\DateTime")


How should I proceed?
thanks in advance

**Comment by spawnia** [comment]
Hey @trasteroandy, thanks for hanging in there!

I just pushed & merged a hotfix, since #320 was just a WIP and merged by accident. I added a test case to the https://github.com/nuwave/lighthouse/blob/master/tests/Integration/Schema/Directives/Fields/HasManyDirectiveTest.php which does show that the global scope is indeed called in this case.

Sorry about that thing with the `@scalar` directive, that regression was fixed in #322. You should be able to try `dev-master` now and see if your Scope is called.

If not, please report back with some more details - or even better: Add a failing test to https://github.com/nuwave/lighthouse/blob/master/tests/Integration/Schema/Directives/Fields/HasManyDirectiveTest.php

Thanks for getting involved.

**Comment by trasteroandy** [comment]
I confirm that it's solved and works perfectly. Very grateful not only for
this fix, but for the excelent package you have implemented.

I appreciate too much the approach led by the scheme, full of goodies.
Best regards from Madrid (Spain)

    Andrés
